src/client/Microsoft.Identity.Client/AppConfig/ApplicationOptions.cs

```diff
@@ -80,6 +80,12 @@ public abstract class ApplicationOptions
         /// </summary>
         public LogLevel LogLevel { get; set; }
 
+        /// <summary>
+        /// Enables you to configure what happens to the log messages. Use this callback to direct MSAL's log messages to your own logging mechanism.
+        /// See https://aka.ms/msal-net-logging
+        /// </summary>
+        public LogCallback LogCallback { get; set; }
```

Comment:
How would this get set via a configuration file?

Reply:
I've seen this used separately, not via configuration file, for example in the MISE code we've been looking toghter at. App developer is confused because they can set the LogLevel but they cannot direct the log messages to their own logging sink.

Reply:
Same argument can be made for `IEnumerable<string> ClientCapabilities` - you need some sort of adapter to transform a space separated string into an array.